### PR TITLE
News site w2

### PR DESCRIPTION
**Pull request by Asivol93**
https://carling-newsite-w2.netlify.app/

News Site - Description
Creating a news site with focus on grid or flexbox and responsive design with mobile first approach.

The problem
I had some issues with starting the project before first live session which made it hard to "clean up" the css code since I wrote a lot of things that needed to be replaced. I had problem with flexbox so I decided to go for grid instead which I thought was easier. I also had some margins and padding on different containers which made my navbar misplaced. I just went through all of my containers one by one and removed those to see what was causing that problem.

What I could have done differently: 
I would have sketched up the project before hand. Instead of creating 1-2 elements and then style them directly. Made it hard to know where in the process I was. If I had more time I would have created subpages for my navbar links and fint a more suitable color scheme and logo that fits with a news site. I believe the design is a little bit to fluffy and cute.

What I'm proud of: 
Im proud of my progress during the week and that I could tackle a lot of problems by googling. I also like that i managed to apply shadow on my pictures within my grid items. Also had some great knowledge sharing with a classmate and my partner (who also is in the boot camp). We met for 5 hours on saturday to help eachother out which was usefull going forward.

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,259 @@
...
+
+.grid-pic img {
+  animation-duration: 6s;
+  animation-iteration-count: infinite;
+  animation-name: logobounce;
+  animation-timing-function: cubic-bezier(0.280, 0.840, 0.420, 1);
+  margin: 10px;
+}
+
+@keyframes logobounce {
```
Comment (MarySnopok) [inline code/style.css:111]: nice effect and it is cool that you managed to add animation , I will also have to try it in the next project.

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,259 @@
...
+   25% {color: #C97C5D;}
+   50% {color: #C89F9C;}
+   75% {color: #EED7C5;}
+   100% {color: #EEE2DF;}
+ }
+
+h2 {
+  font-size: 30px;
+  padding: 2px;
+  height: 30px;
```
Comment (MarySnopok) [inline code/style.css:144]: Checking the page via devtool I found out a tiny bug : when you are viewing the page in mobile version the header sticks to the main banner picture (one with the boat) . So I played around and understood that it is due to the assigned hight of 30px on h2. When you just removing the line 144 : a space appear between the picture and the h2 , what makes it looks consistent with the entire layout. Also other versions (tab and desktop) are not affected with this change and all looks same. Check it out also if you like.

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,259 @@
...
+    padding: 20px;
+    grid-gap: 20px;
+    background: white;
+    height: 400px;
+  }
+  
+  .smallnews-grid {
+    grid-column: span 1;
+    display: grid;
+    flex-direction: column;
```
Comment (MarySnopok) [inline code/style.css:254]: I guess you had played around with different options and forgot to delete this line (254: flex-direction:column) thus its from flex boxes and also if I disable it in dev tools nothing is changing . I suppose the line 252 carries out its function and creates the columns as required. So I suppose 254 can be just deleted.

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,112 @@
...
+        The famous sea mammal has got quite a bad reputation recently for sinking boats, 
+        but from the look of these photos he is getting his sea legs now. We would hop in for a ride if Wally was sailing the ship. 
+        Ahoy there sailor!</p>
+    </div>
+
+  </div>
+
+  <div class="body-section-grid">
+
+    <div class="smallnews-grid">
```
Comment (MarySnopok) [inline code/index.html:63]: occasional use of <article> or <section> could be also a nice decision for a semantic markup and better accessibility of the page content.

**Review by MarySnopok — COMMENTED** [review]
Hi there) . It was a pleasure looking under the hood of your project. I did mine with flexboxes and it was educational to view the Grid version .However it also means that I am not an expect in grid and cant provide any valuable incites on grid layout . I had left few comments in the code , please check them out, might be you will find some of them useful.
Feedback part: 
1) code is working 
2) Blue requirements: 
2.1 page is styled with use of CSS 
2.2 CSS grid is implemented for ensure responsive design
2.3 1 column for mobile , 2 for tablets and 4 for desktop are visible, responsive - requirement met
2.4 The page has header, nav bar, main article  and a grid of news articles.
2.5 hovers effects are applied and working beautifully.
Red level : 
2.6 boxes have shadows, not sure about polaroids effect (someone more skilled in UI/UX has to view to evaluate this point)
2.7 small news content is centered
Black Level:
2.8 Some animations and transitions are applied
Overall I found it to be a working version with all the blue requirements met for sure and some (or might be even all)  of other requirements levels as well . 
There are few suggestions and one bug that I found , Please check the comments in the code for details. PM me if you have any questions.

File: code/index.html [hunk 5]
```diff
@@ -1,22 +1,112 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Nothing+You+Could+Do&display=swap" rel="stylesheet">
+    <title>WWN</title>
+    <link rel="icon" href="./new_logo.png" type="image" sizes="any">
+</head>
+<body>
+  <section class="full-container">
+
+  <header class="header-grid-container">
+    <div class="grid-pic">
+      <img src="./new_logo.png"/>
```
[the remark this answers is not in the document]
Comment (Asivol93) [reply code/index.html:30]: Thank you for reminding me, totally forgot about that!

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,259 @@
...
+  justify-content: center;
+
+}
+
+.body-section-grid img {
+  object-fit: cover;
+  border-radius: 5%;
+  height: 200px;
+  border:2px solid #fff;
+  background: url(img/tiger.png) no-repeat;
```
[the remark this answers is not in the document]
Comment (Asivol93) [reply code/style.css:88]: Ah I tried something but regret it and did not delete it... removed in coming push! Thanks!

File: code/style.css (hunk 2 above)
Comment (Asivol93) [reply code/style.css:144]: Thank you for noticing, I was having issues with this but got lost along the way. This feedback really helped me and I've fixed it.

File: code/style.css (hunk 3 above)
Comment (Asivol93) [reply code/style.css:254]: Your right, I guess I mixed up flexbox and grid when I tried them both. Cleaned it up!

File: code/index.html (hunk 4 above)
Comment (Asivol93) [reply code/index.html:63]: Good point, changed it to article instead, thank you @MarySnopok

File: code/index.html [hunk 7]
```diff
@@ -1,22 +1,112 @@
...
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Rampart+One&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Source+Sans+Pro:wght@300&display=swap" rel="stylesheet">
+    <!--Title font: "Averia Serif Libre"-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
[the remark this answers is not in the document]
Comment (Asivol93) [reply code/index.html:16]: Tags are now on the same block level, not sure what you mean with the first comment tho. Can you specify the comment about identation? :)

**Review by MarySnopok — APPROVED** [review]
Cool , I see all the changes you have made.

**Pull request closed without merging**